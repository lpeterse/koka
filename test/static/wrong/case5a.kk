// case sensitivity
import modD = static/modA
import static/modB

alias wrong = modD/aInt
