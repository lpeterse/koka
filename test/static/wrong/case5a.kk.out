test/static/wrong/case5a.kk(5,15): error: type aInt should be cased as aint